README.md

```diff
@@ -60,6 +60,22 @@ when Ecto gets more complex field selection support we will go further to only q
 
 You will need to handle filtering yourself, the filter is just a map with key=value.
 
+## Configuration
+
+There are several options available through configuration.
```

Comment:
@ybur-yug I think the last sentence I tossed in there should be sufficient for the reasoning so long as we include a link to the spec: http://jsonapi.org/recommendations/